### PR TITLE
Add New Devices: HTC Desire Eye - Samsung Galaxy S8 / S8+ [Exynos & Snapdragon] - YU YUREKA BLACK - Mi4s - Redmi Pro- Zenfone Z00ED

### PR DESCRIPTION
**Pull request by yshalsager**

### CODE REVIEW

File: data/devices/samsung/00_galaxy_s.yml [hunk 1]
```diff
@@ -729,3 +729,192 @@
...
+
+- name: Samsung Galaxy S8+ (Snapdragon)
+  id: dream2qlte
+  codenames:
+    # Regular variant
+    - SM-G955U
+    - dream2qlte
+    - dream2qltezh
+    - dream2qltezhn
+    - lineage_dream2qlte
```
Comment (chenxiaolong) [inline data/devices/samsung/00_galaxy_s.yml]: Is there actually a ROM that uses `lineage_dream2qlte`?

File: data/devices/samsung/00_galaxy_s.yml [hunk 2]
```diff
@@ -729,3 +729,192 @@
...
+    max_brightness: 36500
+    default_brightness: 15000
+    pixel_format: RGBX_8888
+    theme: portrait_hdpi
+
+- name: Samsung Galaxy S8+ (Snapdragon)
+  id: dream2qlte
+  codenames:
+    # Regular variant
+    - SM-G955U
```
Comment (chenxiaolong) [inline data/devices/samsung/00_galaxy_s.yml]: The SM-G955U is the US carrier unlocked version, but that version doesn't have an unlocked bootloader.

Can we limit this to just `dream2qltezh` and `dream2qltezhn`? As far as I know, those are the only Snapdragon S8/S8+ devices that have an unlocked bootloader.

**Comment by chenxiaolong** [comment]
@yshalsager Thanks for the PR! I just have a few questions above.

File: data/devices/samsung/00_galaxy_s.yml (hunk 2 above)
Comment (yshalsager) [inline data/devices/samsung/00_galaxy_s.yml]: Edited !

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Pull request merged**